utils/compatibility/utils.py

```diff
@@ -52,6 +72,21 @@ def write_yaml(file_path, data):
     return False
 
 
+def validate_semver(version_str):
+    try:
+        # Coerce the version string to handle versions like "1.30", "1.27", or "1.2.3"
+        version = semantic_version.Version.coerce(version_str)
+
+        # Check if prerelease and build are empty
+        if version.prerelease or version.build:
+            return False
```

Comment:
`return None` would probably be a slightly more natural typing for this function (shouldn't affect the behavior any though)

Reply:
Ack, that makes sense.